src/app.md

```diff
@@ -268,8 +275,10 @@ fn run() -> Result<()> {
 
     Ok(())
 }
-#
-# quick_main!(run);
+
+fn main(){
```

Comment:
no reason to unhide the error boilerplate.
Also please do not modify the original code if there is no need